packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -86,6 +86,10 @@ export default function Sidebar({
     hasUser: !!user,
     hasFiltered: !alerts?.filter,
   });
+  const hasSquads = squads?.length > 0;
+  const squadsUrl = hasSquads
+    ? `${webappUrl}squads/discover/my`
```

Comment:
We can probably reuse the paths from `squadCategoriesPath`.

Reply:
updated